src/left-nav.jsx

```diff
@@ -39,62 +36,11 @@ const LeftNav = React.createClass({
     docked: React.PropTypes.bool,
 
     /**
-     * A react component that will be displayed above all the menu items.
-     * Usually, this is used for a logo or a profile image.
-     */
-    header: deprecated(React.PropTypes.element,
-      'Instead, use composability.'),
-
-    /**
-     * Class name for the menuItem.
-     */
-    menuItemClassName: deprecated(React.PropTypes.string,
-      'It will be removed with menuItems.'),
-
-    /**
-     * Class name for the link menuItem.
-     */
-    menuItemClassNameLink: deprecated(React.PropTypes.string,
-      'It will be removed with menuItems.'),
-
-    /**
-     * Class name for the subheader menuItem.
-     */
-    menuItemClassNameSubheader: deprecated(React.PropTypes.string,
-      'It will be removed with menuItems.'),
-
-    /**
-     * JSON data representing all menu items to render.
-     */
-    menuItems: deprecated(React.PropTypes.array,
-      'Instead, use composability.'),
-
-    /**
-     * Fired when a menu item is clicked that is not the
-     * one currently selected. Note that this requires the `injectTapEventPlugin`
-     * component. See the "Get Started" section for more detail.
-     */
-    onChange: deprecated(React.PropTypes.func,
-      'It will be removed with menuItems.'),
-
-    /**
-     * Fired when the component is opened.
-     */
-    onNavClose: deprecated(React.PropTypes.func,
-      'Instead, use onRequestChange.'),
-
-    /**
-     * Fired when the component is closed.
-     */
-    onNavOpen: deprecated(React.PropTypes.func,
-      'Instead, use onRequestChange.'),
-
-    /**
      * Callback function that is fired when the open state of the `LeftNav` is
      * requested to be changed. The provided open argument determines whether
      * the `LeftNav` is requested to be opened or closed. Also, the reason
      * argument states why the `LeftNav` got closed or opend. It can be either
-     * `'clickaway'` for menuItem and overlay clicks, `'escape'` for pressing the
+     * `'clickaway'` for overlay clicks, `'escape'` for pressing the
      * escape key and 'swipe' for swiping. For opening the reason is always `'swipe'`.
```

Comment:
Nit: Not your fault, but just noticed 'swipe' on this line also doesn't have the `'swipe'` back ticks. Maybe we can just get it fixed here.